packages/client/mutations/toasts/mapTeamsLimitReminderToToast.ts

```diff
@@ -0,0 +1,43 @@
+import graphql from 'babel-plugin-relay/macro'
+import {Snack} from '../../components/Snackbar'
+import {Threshold} from '../../types/constEnums'
+import {OnNextHistoryContext} from '../../types/relayMutations'
+import makeDateString from '../../utils/makeDateString'
+import {mapTeamsLimitReminderToToast_notification} from '../../__generated__/mapTeamsLimitReminderToToast_notification.graphql'
+import SendClientSegmentEventMutation from '../SendClientSegmentEventMutation'
+
+graphql`
+  fragment mapTeamsLimitReminderToToast_notification on NotifyTeamsLimitReminder {
+    id
+    scheduledLockAt
+    organization {
+      id
+      name
+    }
+  }
+`
+
+const mapTeamsLimitReminderToToast = (
+  notification: mapTeamsLimitReminderToToast_notification,
+  {history, atmosphere}: OnNextHistoryContext
+): Snack => {
+  const {id: notificationId, organization, scheduledLockAt} = notification
+  const {name: orgName, id: orgId} = organization
+
+  return {
+    autoDismiss: 0,
+    key: `newNotification:${notificationId}`,
+    message: `"${orgName}" is over the limit of ${
+      Threshold.MAX_STARTER_TIER_TEAMS
+    } Free Teams. Your free access will end on ${makeDateString(scheduledLockAt)}`,
+    action: {
+      label: 'Upgrade',
+      callback: () => {
+        history.push(`/me/organizations/${orgId}`)
+        SendClientSegmentEventMutation(atmosphere, 'Clicked teams limit reminder snackbar')
```

Comment:
-1 Can we change this to a `Snackbar Clicked` event with a `snackbarType: 'teamLimits'` property?

Reply:
@tghanken we have 2 possible actions on a snackbar.

1) You can click on it anywhere and it will be dismissed
2) You can click on the upgrade button, you will be redirected to the upgrade page and snackbar will be dismissed

I guess in this case we need 3 events
`Upgrade CTA Viewed` and `Upgrade CTA Clicked` and `Snackbar Clicked` ?

Reply:
@tghanken refactored events a bit, can you check?